packages/labs/virtualizer/README.md

```diff
@@ -285,6 +285,65 @@ export class MyItems extends LitElement {
 }
 ```
 
+## ResizeObserver dependency
+
+Virtualizer depends on the standard [`ResizeObserver`]() API, which is supported in all modern browsers. In case your browser support matrix includes older browsers that don't implement `ResizeObserver`, the Virtualizer package includes a `ResizeObserver` polyfill that is known to be compatible with Virtualizer.
+
+### Using the default loader
+
+The package also includes a simple mechanism for loading the `ResizeObserver` polyfill. This mechanism uses feature detection to determine whether the polyfill is required; if it is, then the polyfill is automatically loaded and provided directly to Virtualizer.
+
+You need to invoke the loader—which is aysnchronous—and await its return before running any code that would cause a virtualizer to be instantiated. The simplest way to do this is to load virtualizer itself with a dynamic `import()` statement. For example:
```

Comment:
small spelling error here, should be `asynchronous`

Reply:
Fixed, thanks!